cmd/start.go

```diff
@@ -101,6 +107,8 @@ func initiateServer(ch <-chan os.Signal) error {
 		rpcServ.Start(cfg.Vatz.RPCInfo.Address, cfg.Vatz.RPCInfo.GRPCPort, cfg.Vatz.RPCInfo.HTTPPort)
 	}()
 
+	initPrometheus(promPort, cfg.Vatz.ProtocolIdentifier)
```

Comment:
As I remember, `initMetricServer` could be more general on the Cosmos env.
Because when we call "Prometheus", people usually imagine the Prometheus server, which gathers the metrics actively.

What do you think?

Reply:
I think it would be better to revise it so that it doesn't cause any confusion. Thank you for a good idea.

> Commit: bd4e117b477109bae7925c6a22afc568cbb2e47b